docker-compose-deploy.yaml

```diff
@@ -32,3 +33,46 @@ services:
     ports:
       - "5004:5004"
 
+  grafana:
+    build:
+      context: ./infra/grafana/
+      dockerfile: Dockerfile
+    container_name: grafana
+    ports:
+      - "3000:3000"
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+    volumes:
+      - grafana-storage:/var/lib/grafana
+
+  loki:
+    build:
+      context: ./infra/loki/
+      dockerfile: Dockerfile
+    container_name: loki
+    command: -config.file=/etc/loki/local-config.yaml
+    volumes:
+      - loki-data:/loki
+
+  grafana-alloy:
+    build:
+      context: ./infra/alloy/
+      dockerfile: Dockerfile
+    container_name: alloy
+    environment:
+      - ALLOY_API_KEY=${ALLOY_API_KEY}
+    env_file:
+      - './.env.example'
```

Comment:
**issue (blocking):** Let's not use default credentials on production envs. ~Those in particular don't seem to matter much, but someone may get surprised when their example envs land on prod.~ Those credentials can be used to access open ports, they should be a secret, and ports should be accessible only internally.